recipes/scotch/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "scotch" %}
+{% set version = "6.0.4" %}
+{% set md5 = "d58b825eb95e1db77efe8c6ff42d329f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+  url: http://gforge.inria.fr/frs/download.php/file/34618/{{ name }}-{{ version }}.tar.gz
+
+build:
+  skip: true  # [win]
+  number: 0
+
+requirements:
+ build:
+  - toolchain
+  - zlib 1.2.*  # [unix]
+  - bison
+  - flex
+ run:
+  - zlib 1.2.*  # [unix]
+
+test:
+  commands:
```

Comment:
There seems to be a bunch of executables that are installed. Can we please run `--help` or `--version` or similar to check that they do run without crashing?